app/javascript/components/explore/CellFilteringPanel.jsx

```diff
@@ -31,6 +30,244 @@ export function CellFilteringPanelHeader({
   )
 }
 
+/**
+ * Very brief notes on terms in SCP metadata convention, derived from:
+ * https://singlecell.zendesk.com/hc/en-us/articles/360060609852-Required-Metadata
+ */
+const conventionalMetadataGlossary = {
```

Comment:
I was going to say I worry this could get out-of-date, but in retrospect it has the same risk of that as any other user-facing text in the portal.  So likely not a large issue.